rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -331,6 +380,163 @@ def calculate_claimed_airdrops(
         ).fetchall()
 
 
+def process_airdrop_with_api_data(
+        addresses: Sequence[ChecksumEvmAddress],
+        airdrop_data: AirdropAPIData,
+        protocol_name: str,
+        airdrops_without_decoder: set[str],
+        tolerance_for_amount_check: FVal,
+) -> tuple[list[AirdropClaimEventQueryParams], dict[ChecksumEvmAddress, dict]]:
+    """Returns information for airdrops that can be queried only using remote APIs"""
```

Comment:
Can you better describe input args and output args? Wonder why you need the temp airdrop tuple

Reply:
The temp airdrop tuple is used later to search history events that match it and then decide if the airdrop was claimed. Is part of the old logic and when breaking the code to have it modularized I haven't touch it